### ISSUE TITLE
[Bug] insomnia-plugin-default-headers - Cannot copy as curl with headers

### ISSUE THREAD

**Issue by Yamilquery** [getinsomnia/insomnia#749]
[Bug] – When I try to "Copy as curl" a query, I get my curl incomplete without the headers that I used on my environment vars 

- Insomnia Version: ___ Version 5.13.0 (5.13.0)
- Operating System: ___ OS X High Sierra 10.13.1

## Details

```json
{
"DEFAULT_HEADERS": {
		"Authorization": "Bearer eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJVc2VyOjEiLCJleHAiOjE1MjA4MDQyMjYsImlhdCI6MTUxODIxMjIyNiwiaXNzIjoiQmFyayIsImp0aSI6IjVhYjgyMDJhLTRkYWItNGNiYS05ZjAwLThmNGMxMGRiYTllNSIsInBlbSI6e30sInN1YiI6IlVzZXI6MSIsInR5cCI6eyJzb21lIjoiY2xhaW0ifX0.eAi8i5fnWmI2_59yBC3Pch9OoCWBRbC3YLv3WpqbUZpqNlbj-Q-y92PlGGj_lsP2mtjFClbvHiSLKq3_9yhmRg",
		"Connection": "close",
		"Content-Type": "application/json"
	}
}
```

<img width="1059" alt="screen shot 2018-02-09 at 15 50 11" src="https://user-images.githubusercontent.com/6020065/36051870-34f1e74e-0db1-11e8-809d-06a3e8deb673.png">

Curl returned: 

```
curl --request POST \
  --url http://localhost/api \
  --data '{"query":"subscription AttentionRouteCreated($attention_id: Int!) {\n  attention_route_created(attention_id: $attention_id) {\n    id,\n    attention { id },\n    lat,\n    lng,\n    inserted_at,\n    updated_at\n  }\n}","variables":{"attention_id":17},"operationName":"AttentionRouteCreated"}'
```

**Comment by gschier** [comment]
Thanks for reporting this @Yamilquery! Should be an easy fix

**Comment by aldaris** [comment]
Which version of the app is this fixed in? I'm seeing the same issue with 5.14.9.1895

**Comment by gschier** [comment]
Will be released soon
